### ISSUE TITLE
To read docs, the wiki page about the Report Conversion Utility must be written

### ISSUE THREAD

**Issue by craigfowler** [csf-dev/CSF.Screenplay#173]
There is a wiki page about [the report conversion utility](https://github.com/csf-dev/CSF.Screenplay/wiki/ReportConversionUtility), but it is yet to be written.

**Comment by Griizly** [comment]
Hi, any news regarding this issue ?

**Comment by craigfowler** [comment]
I'm still on an indefinite hiatus from all of my spare-time projects.  In this case, though, I have edited the wiki page and put the basic usage info on it, along with pointing out where the code for this lives.  You'll see it's very basic at the moment, only needing two parameters.

**Comment by craigfowler** [comment]
*I have just noticed/realised that I had the wiki permissions set incorrectly.*  Nobody would have been able to contribute to it except myself.

I have just relaxed them, so that others may contribute if they wish.  In other news I am just starting to get geared up again for spare-time projects.  In reality it may still be quite some time before I pick them up again in earnest, but I can at least see my return on the horizon now.
